frontend/app/.server/domain/repositories/application-year.repository.ts

```diff
@@ -59,3 +59,37 @@ export class ApplicationYearRepositoryImpl implements ApplicationYearRepository
     return applicationYearResults;
   }
 }
+
+@injectable()
+export class MockApplicationYearRepository implements ApplicationYearRepository {
+  private readonly log: Logger;
+
+  constructor(@inject(TYPES.factories.LogFactory) logFactory: LogFactory) {
+    this.log = logFactory.createLogger('DefaultBenefitRenewalRepository');
```

Comment:
```suggestion
    this.log = logFactory.createLogger('MockApplicationYearRepository');
```

By convention `ApplicationYearRepositoryImpl` can also be renamed to `DefaultApplicationYearRepository`